router/router.go

```diff
@@ -84,6 +84,9 @@ func NewRouter(config Config, name PeerName, nickName string) *Router {
 	return router
 }
 
+// Start listening for packets from containers, TCP connections and
```

Comment:
Shouldn't really mention 'containers' here; the router is largely unaware of them.